bcda/cclf/cclf.go

```diff
@@ -253,6 +253,12 @@ func importCCLF(fileMetadata *cclfFileMetadata, importFunc func(uint, []byte, *g
 }
 
 func getCCLFArchiveMetadata(filePath string) (cclfFileMetadata, error) {
+	refDateString := os.Getenv("CCLF_REF_DATE")
```

Comment:
Just a though...
From a configuration simplification perspective, I feel like `CCLF_REF_DATE` should remain static at `time.Now()` and the lookback value (45, in this case) should be the config value.  In other words, we _always_ reference the date comparison against current time, but the lookback value could potentially change from the default value of 45 if we need to.

Reply:
Wouldn’t that be more complicated?  To use our test files in the repo, the lookback value would need to increase over time.

Reply:
Yup, good point.  I didn't think about the test files.